src/main/java/de/muenchen/dave/domain/pdf/templates/messstelle/GangliniePdf.java

```diff
@@ -0,0 +1,54 @@
+package de.muenchen.dave.domain.pdf.templates.messstelle;
```

Comment:
Nenn das file hier bitte um auf GanglinieMessstellePdf, damit in den files aus 
`final de.muenchen.dave.domain.pdf.templates.messstelle.GangliniePdf bean` `(final GanglinieMessstellePdf bean` wird

Reply:
Die Unterscheidung der Files ist ja schon in der Packagestruktur hinterlegt, daher würde ich da keinen Suffix mehr dranhängen wollen. Da wir ja jetzt LCM machen würde ich vorschlagen wir splitten den vorhandenen Service sauber auf und trennen nach Messstelle und Zählstelle. Dann haben wir auch keine Imports mehr von Files mit identischem Namen

Reply:
passt für mich, dann machen wir es im LCM teil